### PR TITLE
Moved IAA spawn to their office.

### PR DESCRIPTION
**Pull request by Kaedwuff**
The IAA spawns were moved to their office chairs.

I also removed the chairs in the security briefing room that they previously occupied. There is now a bit of imposing space between the HoS and the rest of his team, which I like.

### CODE REVIEW

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -1344,9 +1344,6 @@
 /turf/simulated/floor/tiled,
 /area/security/vacantoffice2)
 "acK" = (
-/obj/structure/bed/chair/office/dark{
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm]: It seems you have deleted a chair in the IA office.
Intentional ?

Comment (Arrow768) [reply maps/aurora/aurora-4_mainlevel.dmm]: Actually, never mind. its mentioned in the title.

**Comment by Kaedwuff** [comment]
I guess this didn't make it into the march goal. I thought it had. 

Anyone want to review it?

**Comment by Arrow768** [comment]
There seems to be an issue with mapmerge 26k changes

**Comment by LordFowl** [comment]
I don't really see why this is marked as Bypass Development @Lohikar 
But in that case it does not need to be merged according to dev cycles.

However did you mapmerge properly?

**Comment by Lohikar** [comment]
@LordFowl It was marked bypass dev as it is targeting master. 

Large number of changes might be a result of the resolver tool, though not optimal.

**Comment by LordFowl** [comment]
In that case @Kaedwuff Is there any reason this is targeting master instead of dev? Master is typically reserved for bug-fixes.

**Comment by Kaedwuff** [comment]
Mostly because when Lohikar was explaining the process to me, I was pretty sure he told me map changes to to master.  I might have misunderstood?

**Pull request merged**